workspace.go

```diff
@@ -322,6 +346,10 @@ func (s *workspaces) Read(ctx context.Context, organization, workspace string) (
 		return nil, err
 	}
 
+	// durations come over in ms
+	w.ApplyDurationAverage *= time.Millisecond
+	w.PlanDurationAverage *= time.Millisecond
```

Comment:
Is the goal to convert this so that it is seconds, even though the data comes over in ms?

Reply:
Yeah, this got lost in commenting above; I think we should probably leave the duration alone and use what the API provides directly (ms), unless there's some specific reason we should be returning something different from the canonical source (I don't see any?). Any caller can do whatever they want with it. Pity the attribute(s) isn't named `apply-duration-average-ms` for clarity.

Reply:
The underlying type is a `time.Duration` which expects nanoseconds since epoch, but we send over ms. In order to fit the datatype we need to "tell" the `time.Duration` that the values are milliseconds and not nanoseconds

Reply:
> Pity the attribute(s) isn't named apply-duration-average-ms for clarity.

agreed

Reply:
Ahhhhhh got it, whoops, missed that. 🙇🏻‍♂️